drivers/flash/spi_flash_at45.c

```diff
@@ -703,4 +704,4 @@ static const struct flash_driver_api spi_flash_at45_api = {
 		      POST_KERNEL, CONFIG_SPI_FLASH_AT45_INIT_PRIORITY,      \
 		      &spi_flash_at45_api);
 
-UTIL_LISTIFY(DT_NUM_INST(DT_DRV_COMPAT), SPI_FLASH_AT45_INST, _)
+UTIL_LISTIFY(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT), SPI_FLASH_AT45_INST, _)
```

Comment:
```suggestion
DT_INST_FOREACH_STATUS_OKAY(SPI_FLASH_AT45_INST)
```

along with this change above:

```
#define SPI_FLASH_AT45_INST(idx)					     \
```

Reply:
Applied.